addons/addon-base-raas/packages/base-raas-services/lib/data-source/access-strategy/roles-only/filesystem-role-service.js

```diff
@@ -323,20 +323,14 @@ class FilesystemRoleService extends Service {
     // If it is not found then we have done the deallocation logic already
     if (_.isUndefined(fsRoleEntity)) return;
 
-    // If we don't have the study in the fs role entity then we have done the deallocation logic already
-    if (!hasStudy(fsRoleEntity, studyEntity)) return;
-
-    // Do we have the member account listed in the trust document?
-    if (!hasMemberAccount(fsRoleEntity, memberAccountId)) return;
-
```

Comment:
Add a test for this branch? This would including creating the fsRoleEntity such that it doesn't have study and member account associated.

Reply:
Done